### PR TITLE
cmd: fix err check

### PR DESCRIPTION
**Pull request by Tjianke**
### What problem does this PR solve?

Issue Number: #22979  

Problem Summary:

### What is changed and how it works?

What's Changed: added err check

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

- No release note

### CODE REVIEW

**Comment by bb7133** [comment]
PTAL @tiancaiamao

File: cmd/ddltest/ddl_test.go [hunk 1]
```diff
@@ -304,7 +304,10 @@ func (s *TestDDLSuite) startServer(i int, fp *os.File) (*server, error) {
 		}
 		log.Warnf("ping addr %v failed, retry count %d err %v", addr, i, err)
 
-		db.Close()
+		err = db.Close()
+		if err != nil {
+			panic(err)
```
Comment (zimulala) [inline cmd/ddltest/ddl_test.go:309]: I  think we can print the log and break it here.

Comment (Tjianke) [reply cmd/ddltest/ddl_test.go]: Done.

**Pull request merged**